Sources/Core/ARM/SoftwareBreakpointManager.cpp

```diff
@@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
   }
 #endif
 
+  opcode.clear();
```

Comment:
I just don't understand why we need it. We didn't have it before, right?

On Fri, Aug 12, 2016 at 6:15 PM, Francis Ricci <notifications@github.com>
wrote:

> In Sources/Core/ARM/SoftwareBreakpointManager.cpp
> <https://github.com/facebook/ds2/pull/422#discussion_r74678529>:
>
> > @@ -131,6 +117,8 @@ void SoftwareBreakpointManager::getOpcode(uint32_t type,
> >    }
> >  #endif
> >
> > +  opcode.clear();
>
> I can split into a separate commit if you want?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/422/files/50a5cbdc6a795f7383c093429619b7c05ce45b12..7cdfc54368119ccdfb434303997ba7f05611c54e#r74678529>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbLS_f9TsywygOcuZnHNzgEAvlZmTks5qfRrGgaJpZM4JjTCI>
> .
>



-- 
Stephane Sezer